src/reducers/multicall/feeds.js

```diff
@@ -1,73 +1,44 @@
-import BigNumber from 'bignumber.js';
-
-import { USD } from 'maker';
+import { liquidationRatio } from '@makerdao/dai-plugin-mcd/dist/math';
 import { toHex } from 'utils/ethereum';
-import { fromWei, fromRay, sub, mul, RAY } from 'utils/units';
+import { fromWei } from 'utils/units';
 import {
   ADAPTER_BALANCE,
   MAX_AUCTION_LOT_SIZE,
   LIQUIDATION_PENALTY,
   LIQUIDATOR_ADDRESS,
   LIQUIDATION_RATIO,
-  FEED_SET_USD,
-  FEED_VALUE_USD,
+  DUTY,
   RATE,
-  ILK_RATE,
   LAST_DRIP,
-  PRICE_WITH_SAFETY_MARGIN,
+  SPOT,
   DEBT_CEILING
 } from 'reducers/feeds';
 
-export const priceFeed = addresses => (name, { decimals = 18 } = {}) => ({
-  target: addresses[`PIP_${name}`],
-  call: ['peek()(uint256,bool)'],
-  returns: [
-    [`${name}.${FEED_VALUE_USD}`, val => USD(val, -decimals)],
-    [`${name}.${FEED_SET_USD}`, liveness => (liveness ? 'live' : 'ded')]
-  ]
-});
-
 export const rateData = addresses => name => ({
   target: addresses.MCD_JUG,
   call: ['ilks(bytes32)(uint256,uint48)', toHex(name)],
-  returns: [
-    [
-      `${name}.${RATE}`,
-      val => {
-        const taxBigNumber = new BigNumber(val.toString()).dividedBy(RAY);
-        const secondsPerYear = 60 * 60 * 24 * 365;
-        BigNumber.config({ POW_PRECISION: 100 });
-        return taxBigNumber
-          .pow(secondsPerYear)
-          .minus(1)
-          .toFixed(3);
-      }
-    ],
-    [`${name}.${LAST_DRIP}`]
-  ]
+  returns: [[`${name}.${DUTY}`], [`${name}.${LAST_DRIP}`]]
 });
 
+// FIXME it's redundant to call this for each ilk; once per gem would suffice
 export const ilkVatData = addresses => name => ({
   target: addresses.MCD_VAT,
   call: ['ilks(bytes32)(uint256,uint256,uint256,uint256,uint256)', toHex(name)],
-  returns: [[], [`${name}.${ILK_RATE}`, val => fromRay(val, 5)], [], [], []]
-});
-
-export const pitData = addresses => name => ({
-  target: addresses.MCD_PIT,
-  call: ['ilks(bytes32)(uint256,uint256)', toHex(name)],
   returns: [
-    [`${name}.${PRICE_WITH_SAFETY_MARGIN}`, val => fromRay(val, 5)],
-    [`${name}.${DEBT_CEILING}`, val => fromWei(val, 5)]
+    [],
+    [`${name}.${RATE}`],
+    [`${name}.${SPOT}`],
+    [`${name}.${DEBT_CEILING}`],
+    []
   ]
 });
 
 export const liquidation = addresses => name => ({
-  target: addresses[`MCD_SPOT`],
+  target: addresses.MCD_SPOT,
   call: ['ilks(bytes32)(address,uint256)', toHex(name)],
   returns: [
     [`pip${name}`],
-    [`${name}.${LIQUIDATION_RATIO}`, val => fromRay(mul(val, 100), 0)]
+    [`${name}.${LIQUIDATION_RATIO}`, mat => liquidationRatio(mat)]
```

Comment:
in general we want to store raw values from multicall in the store, so we can use them as input to the new non-OO math functions in the SDK. but in this case it makes sense to store the output of one of these math functions, instead of the raw value, because the other math functions depend only on this output value. I have kept the human-readable name to indicate that.